lib/src/tasks/coverage/api.dart

```diff
@@ -317,13 +407,79 @@ class CoverageTask extends Task {
     return coverage;
   }
 
+  List<int> observatoryPort;
+
   Future _run() async {
     if (_html && !(await platform_util.isExecutableInstalled('lcov'))) {
       _done.completeError(new MissingLcovException());
       return;
     }
 
-    await _collect();
+    TaskProcess serve;
+
+    if(functional_tests.isNotEmpty){
+      //    String executable = 'selenium-server';
+//    int port = await getOpenPort();
+      List args = ["-port"
+//    ,"${port}"
+      ];
+      args = [];
+      observatoryPort = new List<int>();
+      RegExp _serving = new RegExp("Serving .* on http:\/\/localhost:8080");
+      serve = new TaskProcess('pub', ['serve']);
```

Comment:
i think for now this is necessary because currently functional tests need to be run in a separate directory with their own pubspec.yaml (when webdriver.dart and w_webdriver_utils get off of the unittest package I think this might work)